src/tls/openssl/tls.c

```diff
@@ -266,6 +290,7 @@ int tls_alloc(struct tls **tlsp, enum tls_method method, const char *keyfile,
 		}
 	}
 
+	tls_enable_sni(tls);
```

Comment:
should SNI be enabled by default, also for clients ?

Reply:
We can enable it in first call of `tls_add_certf()`.